src/main/g8/src/main/scala/$package$/api/Utils.scala

```diff
@@ -0,0 +1,23 @@
+package $package$.api
+
+import zio._
+import zhttp.http._
+
+import $package$.domain._
+import $package$.api.Extensions._
+
+private[api] object Utils:
+
+  def extractLong(str: String): IO[ValidationError, Long] =
+    ZIO
+      .attempt(str.toLong)
+      .refineToOrDie[NumberFormatException]
+      .mapError(err => ValidationError(err.getMessage))
+
+  def handleError(err: DomainError): UIO[Response] = err match {
+    case NotFoundError          => ZIO.succeed(Response.status(Status.NotFound))
+    case ValidationError(msg)   => msg.toResponseZIO(Status.BadRequest)
+    case RepositoryError(cause) => cause.getMessage.toResponseZIO(Status.InternalServerError)
```

Comment:
Message from this error shouldn't be known to the client. We need some generic message here.

Reply:
Solved!